src/main/flight/pid.c

```diff
@@ -469,7 +469,7 @@ void pidInitConfig(const pidProfile_t *pidProfile)
     // display when it exceeds 25% of its possible range. This gives a useful indication
     // of AG activity without excessive display.
     antiGravityOsdCutoff = 1.0f;
-    if (antiGravityMode == ANTI_GRAVITY_SMOOTH) {
+    if ((antiGravityMode == ANTI_GRAVITY_SMOOTH) && antiGravityEnabled)) {
```

Comment:
This will not work - `pidInitConfig()` is only run on PID setting changes, and we can not rely on the anti gravity switch being active at that time.

Reply:
Understood.